FrontEnd/src/components/BookCard.scss

```diff
@@ -10,19 +22,23 @@
 
     img {
         width: 100%;
-        padding: 5px;
+        height: 300px;
         border-radius: 5px;
-
-        max-width: 210px;
-        max-height: 275px;
+        object-fit: cover;
+        object-position: center;
```

Comment:
I have changed it to work like this to as the covers need to have the same height to ensure they stay in a grid formation and don't go all over the place~ I think it still looks good though?

Reply:
as long as it works ok, I'm totally fine with this =]